src/mcts/search.cc

```diff
@@ -799,6 +801,28 @@ EdgeAndNode Search::GetBestRootChildWithTemperature(float temperature) const {
   return {};
 }
 
+// Returns a random child of a root weighted by move probability.
+EdgeAndNode Search::GetRandomChildbyP() const {
+  // Get sum of weights for roll.
+  float total_weights = 0.0;
+  for (auto& edge : root_node_->Edges()) {
+    total_weights += edge.GetP();
```

Comment:
Is it? I'll have to look into that. If so, then I'll just remove this section.

Reply:
The sum of P is only not 1.0 if there are moves which lead to a forced loss, as these get their policy zeroed. However, this will only happen with some amount of search.

I'd still propose to keep this logic, and use it to introduce a parameter for policy cutoff like suggested in #1734, so absolute blunders can be suppressed.